Libplanet/Address.cs

```diff
@@ -81,18 +83,15 @@ public Address(in ImmutableArray<byte> address)
         /// cref="byte"/> array (i.e., <paramref name="address"/>).
         /// </summary>
         /// <param name="address">An array of 20 <see cref="byte"/>s which
-        /// represents an <see cref="Address"/>.  This must not be <see langword="null"/>.
-        /// </param>
-        /// <exception cref="ArgumentNullException">Thrown when <see langword="null"/> was
-        /// passed to <paramref name="address"/>.</exception>
+        /// represents an <see cref="Address"/>.</param>
         /// <exception cref="ArgumentException">Thrown when the given <paramref
         /// name="address"/> array did not lengthen 20 bytes.</exception>
         /// <remarks>A valid <see cref="byte"/> array which represents an
         /// <see cref="Address"/> can be gotten using <see cref="ToByteArray()"
         /// /> method.</remarks>
         /// <seealso cref="ToByteArray()"/>
         public Address(byte[] address)
-            : this(address?.ToImmutableArray() ?? throw new ArgumentNullException(nameof(address)))
+            : this(address.ToImmutableArray())
```

Comment:
I might be wrong, but couldn't arrays be implicitly null?

Reply:
`#nullable enabled` has been applied in #2778. Although this doesn't syntactically prevent `null` from being passed, if we are to perform null check every time, I don't see the point of having `null`able context. 🙄